kubernetes/client.go

```diff
@@ -91,6 +93,23 @@ func NewClient() (*IstioClient, error) {
 	return &client, nil
 }
 
+func LabelsMatch(serviceLabels, filterLabels map[string]string) bool {
+	labelMatch := true
+
+	filterLabelsKeys := reflect.ValueOf(filterLabels).MapKeys()
+	index := 0
+
+	for labelMatch && index < len(filterLabelsKeys) {
```

Comment:
Why is reflect necessary here? Simple range loop like `for key, value := range filterLabels` wouldn't work?

Reply:
you are right @jotak, I didn't think about using `break` for an early return.